test/Makefile

```diff
@@ -2,6 +2,9 @@
 
 VDSO_DIR := ../target
 VDSO_LIB := $(VDSO_DIR)/vdso64.so
+DEP_DIR := /root/dependency
+VDSO_DEP := $(DEP_DIR)/vdso64.so
```

Comment:
So `target` should be the right directory. 

Previously, we added `/root/dependency` to the `asterinas/asterinas` image to store files for faster compilation process and CI. Do we still need to provide this option in the future? Does it need to be removed in the `asterinas/asterinas` image?

Reply:
Now this PR only fixes missing syscall tests.

Reply:
> So `target` should be the right directory.
> 
> Previously, we added `/root/dependency` to the `asterinas/asterinas` image to store files for faster compilation process and CI. Do we still need to provide this option in the future? Does it need to be removed in the `asterinas/asterinas` image?

Installing the two files in the Docker to avoid downloading is a wise decision, but it is not done properly. Read the Dockerfile more carefully and you will note two things. First, the two `wget` commands should never have appeared in the `build-base` phase; that is not where they belong to. Study our multi-stage Dockerfile more carefully. Second, `/root/dependency` is an example of bad naming. The word "dependency" is too vague. Don't you think everything that is installed in the Docker image is some kind of "dependency"?